### DISCUSSION TITLE
Version 2.5

### DISCUSSION THREAD

**Discussion by vkbo** [vkbo/novelWriter#1973] [General]
### Release Notes

The 2.5 release fixes a number of minor issues from the RC 1 pre-release, and updates most translation files. Polish has also been added.

See the website for complete [Release Notes](https://novelwriter.io/releases/release_2_5.html).

### Detailed Changelog

**Bugfixes**

* Fix an issue with unresponsiveness when switching from Outline View to Tree View with a keyboard shortcut. Issue #1949. PR #1954.
* Fix an issue with switching focus between editor and viewer when the viewer is hidden. Issue #1950. PR #1954.
* Fix an issue with first line indentation where it would sometimes indent first paragraph when it should not. Issue #1965. PR #1971.

**Usability**

* Remove the "Apply" button from Preferences as changing theme does not updated the Preferences dialog, which is unexpected behaviour. The Preferences dialog is now also consistent with other settings dialogs. Issue #1951. PR #1954.
* Improved the error messages when opening a project fails. Issue #1944. PR #1956.
* Make sure the Project Name is selected and in focus when switching to the New Project Form on the Welcome dialog. Issue #1967. PR #1968.

**Improvements**

* Add first line indentation to Manuscript previews. Issue #1961. PR #1962.

**Internationalisation**

* Norwegian and US English translations updated by @vkbo. PR #1955.
* German translation updated by Myian (@HeyMyian). PR #1955.
* Italian translation updated by Lou Cyper (loucyper1). PR #1955.
* Dutch translation updated by Martijn van der Kleijn (mvdkleijn). PR #1955.
* Chinese translation updated by @longqzh. PR #1955.
* Portuguese translation updated by Oli Maia. PR #1955.
* Japanese translation updated by @hebekeg. PR #1955.
* French translation updated by Albert Aribaud (@aaribaud). PR #1955.
* Polish translation added by Anna Maria Polak (@Nauthiz). PR #1955.

**Packaging**

* Rewrote the packaging utilities script. Windows builds are now created in GitHub actions, which is needed for signed packages when this is enabled. PRs #1958 and #1959.

**Documentation**

* Updated the documentation for 2.5. PR #1928.

<hr /><em>This discussion was created from the release <a href='https://github.com/vkbo/novelWriter/releases/tag/v2.5'>Version 2.5</a>.</em>

**Comment by HeyMyian** [comment]
Not detected by 1.415.24.0 or 1.415.25.0 💪 

For the updated German translation I've been in contact with Crowdin user pierre.audite, who gave helpful input on a few strings. Not sure if/how they should/want to be credited.

**Reply by vkbo** [reply]
I removed all URLs in the credits document, which may or may not have helped. At least VirusTotal complained less. Also, it seems Microsoft Defender may have been panicking extra over Inno Setup installer on the last release since I used a version that was only a few days old. Maybe it hadn't seen it before? In any case, I will soon have the Windows installers signed. Got most of it set up.

As for credits for translations, I only credit the original translators and larger contributions or new maintainers. I link to the Crowdin contributor page for the rest.

**Comment by ahoythedinghy** [comment]
Just wanted to say thank you.  I've been using novelwriter for roughly(?) a year now (after trying several other tools first), and I use it daily.  I have been really pleasantly surprised by the pace of constant improvements.  I really love the tool and I'm very grateful for your work.  Every time there is a new release I'm excited to update.
